### PR TITLE
MintNFT type

### PR DESCRIPTION
**Pull request by kugaudo**

### CODE REVIEW

File: zksync_sdk/types/transactions.py [hunk 1]
```diff
@@ -90,6 +90,18 @@ def decimal_str_amount(self, amount: int) -> str:
...
+    id: int
+    symbol: str
+    creator_id: int
+    content_hash: str
+    creator_address: str
+    serial_id: int
+    address: str
+
+    def decimal_amount(self, amount: int) -> Decimal:
+        return Decimal(amount)*Decimal(10).scaleb(-1)
```
Comment (StanislavBreadless) [inline zksync_sdk/types/transactions.py:103]: the number of decimals for nft is always 0, e.g. the nft has no decimal points

File: .gitignore [hunk 2]
```diff
@@ -27,4 +27,5 @@ env/
 venv/
 .env
 .coverage
-.tox/
\ No newline at end of file
+.tox/
+.idea/
```
Comment (StanislavBreadless) [inline .gitignore:31]: Nit: please add a newline at the end of file

**Review by StanislavBreadless — COMMENTED** [review]
Will test it later

**Comment by StanislavBreadless** [comment]
Also please make sure that the CI passes

File: tests/test_wallet.py [hunk 3]
```diff
@@ -76,6 +76,12 @@ async def test_forced_exit(self):
 
         assert tr
 
+    async def test_mint_nft(self):
```
Comment (ly0va) [inline tests/test_wallet.py:79]: what about `test_withdraw_nft`?

File: zksync_sdk/serializers.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 FEE_EXPONENT_BIT_WIDTH = 5
 FEE_MANTISSA_BIT_WIDTH = 11
 MAX_NUMBER_OF_ACCOUNTS = 2 ** 24
-MAX_NUMBER_OF_TOKENS = 128
+MAX_NUMBER_OF_TOKENS = 4294967295
```
Comment (ly0va) [inline zksync_sdk/serializers.py:8]: Probably better to specify this explicitly as `2 ** 32 - 1`

**Review by ly0va — CHANGES_REQUESTED** [review]
Also, what about [full exit for NFT](https://github.com/matter-labs/zksync/blob/1387e44b22f794910867132b361ac12222691268/sdk/zksync.js/src/wallet.ts#L1388)?

**Comment by StanislavBreadless** [comment]
> Also, what about [full exit for NFT](https://github.com/matter-labs/zksync/blob/1387e44b22f794910867132b361ac12222691268/sdk/zksync.js/src/wallet.ts#L1388)?

Good point! This method is not needed for initial development. We'll add this in the next iteration.

File: tests/test_wallet.py (hunk 3 above)
Comment (kugaudo) [reply tests/test_wallet.py:79]: Working on it

**Review by ly0va — APPROVED** [review]

**Pull request merged**